app/users/resources.py

```diff
@@ -0,0 +1,95 @@
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, User
+from app.lib.response_util import buildOkResponse
+
+
+def returnUser(user):
+    """Private method to convert a user model into a dictionary.
+
+        Args:
+        user: A user model.
+    """
+
+    return {
+        "id": user.id, "name": user.name
+    }
+
+
+class UserListResource(Resource):
+    """Class for creating and accessging users."""
+    def get(self):
+        """
+            This method gets a list of all the users.
+
+            Returns:
+            Method returns the a list of user names and ids in dictionary
+            format.
+        """
+        users = User.query.all()
+        return buildOkResponse([returnUser(user) for user in users])
+
+    def post(self):
+        """This method adds a new user.
+
+            Returns:
+            Method returns the new user name and id in dictionary format.
```

Comment:
Nit: Indent one over and perhaps expand the description. `Returns the newly created user's name and id in dictionary format in an OK response.`